src/web/components/SignInGate/gateDesigns/shared.tsx

```diff
@@ -62,7 +54,12 @@ export const actionButtons = css`
 	margin-bottom: 42px;
 
 	> a {
-		margin-right: ${space[9]}px !important;
+		margin-right: ${space[4]}px !important;
+
+		${from.mobileMedium} {
+			margin-right: ${space[9]}px !important;
+		}
+
```

Comment:
fixes an issue where at 320px viewport there would be scroll and overflow on x due to the sign in gate buttons